VTEX - Logistics API.json

```diff
@@ -2130,10 +2131,14 @@
                                                 "items": {
                                                     "type": "array",
                                                     "description": "Geolocation coordinates, first longitude than latitude.",
+                                                    "example": [
+                                                            -34.82939147949219,
+                                                            -7.115228652954102
+                                                        ],
                                                     "items": {
                                                         "type": "number",
                                                         "description": "Longitude and latitude.",
```

Comment:
I see the item can be one or another, but they always come in pairs, so it is also valid to say "and" instead of "or". Besides it seems more accurate mentioning them in the order they appear, so I will keep it as it is.

Reply:
This comment was in the sense that, within this array, an item can be either the longitude or a latitude. the first item of the array will always be the longitude, and the second item will be the latitude. When describing the schema, you should describe the information contained in an item (an element of the array), not the information contained in the array.